packages/react-instantsearch-core/src/connectors/connectToggleRefinement.js

```diff
@@ -62,20 +64,48 @@ export default createConnector({
   displayName: 'AlgoliaToggle',
 
   propTypes: {
-    label: PropTypes.string,
+    label: PropTypes.string.isRequired,
+    attribute: PropTypes.string.isRequired,
+    value: PropTypes.any.isRequired,
     filter: PropTypes.func,
-    attribute: PropTypes.string,
-    value: PropTypes.any,
     defaultRefinement: PropTypes.bool,
   },
 
-  getProvidedProps(props, searchState) {
+  getProvidedProps(props, searchState, searchResults) {
+    const { attribute, value } = props;
+    const results = getResults(searchResults, this.context);
     const currentRefinement = getCurrentRefinement(
       props,
       searchState,
       this.context
     );
-    return { currentRefinement };
+
+    const facetValues = results && results.getFacetValues(attribute);
+    const facetValue =
```

Comment:
These two variable names are confusing.

Reply:
Any suggestions? I'm didn't find something else.

Reply:
When I deal with both singular and plural variable names, I tend to go for `allFacetValues` and `facetValue`. WDYT?

Reply:
To be honest I don't see a huge benefit, I'm used to plural singular like the original one. But I get your point.

Reply:
When I came across these two lines I thought you were referring to the same variable. The benefit is just to avoid this confusion. This is subjective though.

Reply:
Yep I've renamed it.